lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -544,9 +544,42 @@ void TypeLoweringVisitor::visitDecl(WireOp op) {
     std::string loweredName = "";
     if (!name.empty())
       loweredName = name + field.suffix;
+    SmallVector<Attribute> loweredAttrs;
+    // For all annotations on the parent op, filter them based on the target attribute.
+    for (auto opAttr : op.annotations()){
+      auto di = opAttr.dyn_cast<DictionaryAttr>();
+      if (!di) {
+        loweredAttrs.push_back(opAttr);
+        continue;
+      }
+      auto targetAttr = di.get("target");
+      if (!targetAttr) {
+        loweredAttrs.push_back(opAttr);
+        continue;
+      }
+
+      ArrayAttr subFieldTarget = targetAttr.cast<ArrayAttr>() ;
+      std::string targetStr = "";
+      for (auto fName: subFieldTarget){
+        std::string fNameStr = fName.cast<StringAttr>().getValue().str();
+        // The fNameStr will begin with either '[' or '.', replace it with an '_' to construct the suffix.
+        fNameStr[0] = '_';
+        // If it ends with ']', then just remove it.
+        if (fNameStr.back() == ']')
+          fNameStr.erase(fNameStr.size()-1);
+
+        targetStr += fNameStr;
+      }
+      if (!targetStr.empty()){
+        if (field.suffix.find(targetStr) != std::string::npos)
+          loweredAttrs.push_back(opAttr);
```

Comment:
If this succeeds, it should strip the `target` key/value pair. I don't think we need to keep that around once the attribute is correctly applied.

Reply:
Done, removed the ``target`` attribute